docs/csharp/programming-guide/concepts/async/snippets/index/AsyncBreakfast-final/Program.cs

```diff
@@ -18,7 +18,7 @@ static async Task Main(string[] args)
 
             // <SnippetAwaitAnyTask>
             var allTasks = new List<Task>{eggsTask, baconTask, toastTask};
-            while (allTasks.Any())
+            while (allTasks.Count > 1)
```

Comment:
Looks like it doesn't wait for the last task. Should it compare with zero?

Reply:
```suggestion
            while (allTasks.Count > 0)
```